packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -144,12 +144,9 @@ const ModalContent = forwardRef((props, ref) => {
   delete customProps.closePortal;
   delete customProps.fallbackFocus;
 
-  const platformIsiOS = !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);
-
   const modalContent = (
     <div
       {...customProps}
-      tabIndex={platformIsiOS || isCalledFromNotificationDialog ? '-1' : '0'}
```

Comment:
Does container receive initial focus  without tabIndex ?

Reply:
Yes, the primary focus is on the container.PFA

https://github.com/cerner/terra-framework/assets/126655256/88bc9479-2c09-4dcc-be88-05b2309e550a